src/csharp/Pester/Test.cs

```diff
@@ -18,11 +18,15 @@ public Test()
             PluginData = new Hashtable();
             ErrorRecord = new List<object>();
 
-            var runtime = new Hashtable();
-            runtime.Add("Phase", null);
-            runtime.Add("ExecutionStep", null);
-            FrameworkData = new Hashtable();
-            FrameworkData.Add("Runtime", runtime);
+            var runtime = new Hashtable
+            {
+                { "Phase", null },
+                { "ExecutionStep", null }
```

Comment:
```suggestion
                ["Phase"] = null,
                ["ExecutionStep"] = null,
```

not 100% sure this works for hashtables.